traffic_ops/toclientlib/toclientlib.go

```diff
@@ -808,6 +809,7 @@ type ReqInf struct {
 	CacheHitStatus CacheHitStatus
 	RemoteAddr     net.Addr
 	StatusCode     int
+	Headers        http.Header
```

Comment:
Should these be called `ResponseHeaders` as opposed to just `Headers`? I think these could be confused as request headers, since the type name is short for "request info".

Reply:
Changed